libraries/libfc/CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
 add_subdirectory( secp256k1 )
 
 set( WITH_PROCPS OFF CACHE BOOL "" FORCE)
-set( CURVE "ALT_BN128" CACHE STRING "" FORCE)
+# set( CURVE "ALT_BN128" CACHE STRING "" FORCE)
 set( USE_ASM OFF CACHE BOOL "" FORCE)
 set( IS_LIBFF_PARENT OFF CACHE BOOL "" FORCE)
 set( FF_INSTALL_COMPONENT "${FC_INSTALL_COMPONENT}")
-add_subdirectory( libraries/ff )
+# add_subdirectory( libraries/ff )
```

Comment:
There are a number of commented out lines in this PR. They need to be removed or otherwise taken care of.

This entire cmake block can also be removed, from line 3-8